example/config/vanti-ugs/ui-config.json

```diff
@@ -22,6 +22,8 @@
         "clientId": "scos-opsui"
       }
     },
+    "wasteRecordsSource" : "van/uk/brum/ugs/devices/waste",
+    "wasteRecordUnit" : "kg",
```

Comment:
As the waste page is in the ops section in the ui, maybe these properties should exist under `ops.waste` instead of at the top level in this config?